ui-tests/src/test/java/io/syndesis/qe/steps/integrations/IntegrationSteps.java

```diff
@@ -355,4 +355,10 @@ public void waitForIntegrationPodShutdown(String integartionPodName) throws Inte
                 OpenShiftWaitUtils.areNoPodsPresent(integartionPodName),1000,5 * 60 *1000);
     }
 
+    @And("^Camilla drags exported integration \"([^\"]*)\" file to drag and drop area$")
+    public void importIntegrationViaDragAndDrop(String integrationName) throws InterruptedException {
+        assertTrue("Import of integration failed.", listComponent.importIntegrationViaDragAndDrop(integrationName));
```

Comment:
Let's switch these asserts to Assertj variant. I'd like to refactor everything eventually. But for now keed the newcomers fresh :)